x-pack/plugins/lens/public/lens_suggestions_api.ts

```diff
@@ -63,7 +63,11 @@ export const suggestionsApi = ({
   });
   if (!suggestions.length) return [];
   const activeVisualization = suggestions[0];
-  // compute the rest suggestions depending on the active one
+  // remove the lnsMetric check when new metric is on GA and hide will be false
+  if (activeVisualization.visualizationId !== 'lnsMetric' && activeVisualization.hide) {
```

Comment:
this is a new check as well ? should we add something to pr description about it ?

Reply:
This is a new check yes, will add to description!